### DISCUSSION TITLE
why webpack change my require path. ("version": "5.42.1")

### DISCUSSION THREAD

**Discussion by 929853209** [webpack/webpack#14153] [Q&A]
I get an error in my program (webpack: "version": "5.42.1"):

![image](https://user-images.githubusercontent.com/17702813/132180135-cfe0869c-ea19-4006-b895-4c637eba7759.png)

At first I think it's Axios's problem, but then I found this pkg is my local package.json

（webpack://myProgram/./node_modules/axios/lib/helpers/validator.js?f9a6）
![image](https://user-images.githubusercontent.com/17702813/132180593-8d298da6-17ed-4ed2-96d9-683ebe57183a.png)

It should be Axios's  pkg with the path.  And then I found this code in Call Stack:

![image](https://user-images.githubusercontent.com/17702813/132180908-12df0d62-bbda-4af8-b80e-25e06f422902.png)

It seems be Webpack changed my require path from `./../../package.json` to `./package.json`, and with the flat node_modules ,it get my local package.json.

Can someone help me explain why Webpack changed my require path? And is this a BUG? thx very much。

**Comment by sokra** [comment]
Thanks for your report. It would be great if you reduce your issue to a small reproducible example. Best put this example into a github repository together with instructions how to get to the problem.
